superset-frontend/src/explore/components/DataTablesPane/index.tsx

```diff
@@ -152,14 +155,19 @@ const DataTable = ({
   }
   if (data) {
     if (data.length === 0) {
-      return <span>No data</span>;
+      return (
+        <EmptyStateMedium
+          image="document.svg"
+          title={t(`No ${type} were returned for this query`)}
```

Comment:
I think we need to translate this like this:
```suggestion
          title={t(`No %s were returned for this query`, type)}
```

Reply:
Actually, we probably need to translate these like this to make sure the translators have the necessary context to fully translate the type strings:

```typescript
if (type === 'sample') {
  return t(`No samples were returned for this query`)
}
if (type === 'result') {
  return t(`No results were returned for this query`)
}
```
I think this getting this right is pretty important, as these will be very visible fixtures in the application.

Reply:
Done